inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -6,11 +6,31 @@
 
 #include "single_layer_tests/convolution.hpp"
 #include "common_test_utils/test_constants.hpp"
+#include "../skip_tests_check.hpp"
 
 using namespace LayerTestsDefinitions;
 
 namespace {
 
+class GnaConvolutionLayerTest : public ConvolutionLayerTest, GnaLayerTestCheck {
```

Comment:
Consider a template class instead of GnaConvolutionLayerTest and GnaConvolutionReluSequenceTest.

template <class T>
class GnaSkipable : public T, GnaLayerTestCheck { ...}

Reply:
This solution causes issues with gtest's TEST_P macro. The reason is that the macro uses its first parameter, which is test class name, as a part of name of a new class being created in the macro. So template class cannot be used as parameter to this macro.